packages/components/src/Divider/Divider.mdx

```diff
@@ -67,7 +67,7 @@ should be used within a [`Content`](/components/content) component.
           </Text>
         </Content>
       </Card>
-      <Divider vertical size="large" />
+      <Divider direction="vertical" size="large" />
```

Comment:
This could be a "fast-follow" but I would prefer that we document examples using the `base` size primarily. I also wouldn't necessarily use a Divider in between two cards as an example, as Cards are typically intended to be their own grouping mechanism, but perhaps content within a Card?

![image](https://user-images.githubusercontent.com/39704901/135903335-60492b76-c26b-4c34-9d08-a5a1928459c2.png)